Resources/Prototypes/_Harmony/Actions/types.yml

```
@@ -10,3 +10,17 @@
       state: icon
     itemIconStyle: NoItem
     event: !type:ToggleActionEvent {}
+
+- type: entity
+  id: DividingGoose
+  name: Divide
+  description: Divde your cells.
```

Minor spelling mistake
```suggestion
  description: Divide your cells.
```